### DISCUSSION TITLE
Top-of-search IS availability from V3 reporting?

### DISCUSSION THREAD

**Discussion by clancyred** [amzn/ads-advanced-tools-docs#74] [Q&A] [answered]
In the ad console under Campaign --> Ad group --> Targeting, there is a metric for "Top-of-search IS", which is Top-of-search impression share (IS).  

This metric does not appear as an option in the V3 reports documentation:

https://advertising.amazon.com/API/docs/en-us/reporting/v3/metrics

Is this metric available through the reporting API under another name?

**Comment by elizafarley** [comment, marked as answer]
Hi @clancyred, right now impression share metrics are not supported in the version 3 reporting API. This is something the team is working on. Any updates will be posted in the [release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes).
